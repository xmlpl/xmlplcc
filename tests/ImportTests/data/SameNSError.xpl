import std.hello;
import std.hello2;

node[] main() {
}
