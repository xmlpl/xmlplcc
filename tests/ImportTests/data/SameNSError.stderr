./data/SameNSError.xpl:error: Cannot import library '/home/jcoffland/projects/xmlpl/install//lib/xmlpl/std/libhello2.la' with same namespace as '/home/jcoffland/projects/xmlpl/install//lib/xmlpl/std/libhello.la'
./data/SameNSError.xpl:error: input contains errors
